app/main.ipynb

```diff
@@ -126,19 +126,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "subsequence = array[1:6]\n",
+    "\n",
+    "result = subsequence.sum()"
```

Comment:
The subsequence selected from the array does not sum to 27. You need to adjust the indices to select the correct elements that sum to 27.